src/open_inwoner/cms/cases/views/status.py

```diff
@@ -1029,3 +1029,47 @@ def get_context_data(self, **kwargs):
             "cases:case_detail_contact_form", kwargs=self.kwargs
         )
         return context
+
+
+class LegacyCaseDetailHandler(View):
+    """Redirect the legacy case detail to the current version with ZGW API group ref."""
+
+    def get(
+        self,
+        request,
+        object_id: str,
+    ):
+        redirect_url = None
+        match ZGWApiGroupConfig.objects.count():
```

Comment:
If we expect cases where `count > 1` and `count == 0` to be exceptional, then a `try/except` block would be more efficient. It's not a big deal though.